sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssuesWsParameters.java

```diff
@@ -27,7 +27,7 @@
 /**
  * @since 5.3
  */
-public class IssueFilterParameters {
+public class IssuesWsParameters {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this class so it has no more than 20 fields, rather than the 46 it currently has. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1820)